README.md

```diff
@@ -31,9 +31,14 @@ by [Marek Magdziak](https://github.com/mkmarek) and released with a MIT license.
 
 ## Documentation
 
-http://graphql-dotnet.github.io
+1. http://graphql-dotnet.github.io - documentation site that is built from the [docs](/docs2/site/) folder in the `master` branch.
+2. https://graphql.org/learn - learn about GraphQL, how it works, and how to use it.
 
-This site is in sync with `master` branch.
+## Debugging
+
+All packages generated from this repository come with embedded pdb and support [Source Link](https://github.com/dotnet/sourcelink).
```

Comment:
Ping @grean for https://github.com/graphql-dotnet/graphql-dotnet/issues/2257#issuecomment-776213185